app/assets/javascripts/discourse/tests/integration/components/admin-theme-setting-schema-test.gjs

```diff
@@ -133,48 +140,48 @@ module(
         <AdminThemeSettingSchema @schema={{schema}} @data={{data}} />
       </template>);
 
-      const tree = queryRenderedTree();
+      const tree = new TreeFromDOM();
 
-      assert.equal(tree.length, 2);
-      assert.true(tree[0].active, "the first node is active");
-      assert.false(tree[1].active, "other nodes are not active");
+      assert.equal(tree.nodes.length, 2);
```

Comment:
We don't lint against it yet, but `assert.strictEqual()` is preferred over `assert.equal()`